src/Microsoft.VisualStudio.Composition/RuntimeComposition.cs

```diff
@@ -21,12 +21,22 @@ public class RuntimeComposition : IEquatable<RuntimeComposition>
         private readonly IReadOnlyDictionary<string, IReadOnlyCollection<RuntimeExport>> exportsByContractName;
         private readonly IReadOnlyDictionary<TypeRef, RuntimeExport> metadataViewsAndProviders;
 
+        public const string InvalidCompositionException = nameof(InvalidCompositionException);
```

Comment:
This seems like an odd public API to add. That sounds like an exception type, yet it's just a constant named after itself.

Can we keep it private?

Reply:
I see you've already done this. :)